Content.Shared/_CD/CartridgeLoader/Cartridges/VoteLinkUiState.cs

```diff
@@ -0,0 +1,99 @@
+using Robust.Shared.Serialization;
+using Robust.Shared.Timing;
+
+namespace Content.Shared._CD.CartridgeLoader.Cartridges;
+
+[Serializable] [NetSerializable]
```

Comment:
ss14 puts annotations on separate lines, or uses the `[Annotation1, Annotation2]` notation